docs/monitoring/infrastructure.md

```diff
@@ -0,0 +1,49 @@
+title: Sematext Core Infrastructure Monitoring
+description: Sematext Infrastructure Monitoring gives you insight into your whole infrastructure automatically based on the applications, containers, servers and general infrastructure you are monitoring.
+
+
+Points to cover:
+
+## Overview
+
+Sematext Monitoring provides complete insight into your infrastructure - applications, servers, containers, processes, inventory, events...
+
+The centerpiece is newly introduced Infra App which acts as a container for all infrastructural data about your architecture. As with other App types, you can create as many Infra Apps under your account as you wish. Each Infra App would typically be used to gather data from one environment (for example, one Infra App for production environment, one for test environment and one for development environment).
```

Comment:
The centerpiece is **a** * newly introduced Infra App which acts as a container for all infrastructural data about your architecture.

Reply:
all infrastructural data about your architecture => all data about your infrastructural
would typically => is typically
(for example, one Infra App for production environment, one for test environment and one for development environment) => For example, ...... - basically just make it a sentence, no need for parentheses.